src/admin/admin.js

```diff
@@ -0,0 +1,334 @@
+    root: {
+        width: "100%",
+    },
+    paper: {
+        width: "100%",
+        marginBottom: theme.spacing(2),
+    },
+    table: {
+        minWidth: 750,
+    },
+    visuallyHidden: {
+        border: 0,
+        clip: "rect(0 0 0 0)",
+        height: 1,
+        margin: -1,
+        overflow: "hidden",
+        padding: 0,
+        position: "absolute",
+        top: 20,
+        width: 1,
+    },
+}));
+
+export default function OverviewTable() {
+    const classes = useStyles();
+    const [order, setOrder] = React.useState("asc");
+    const [orderBy, setOrderBy] = React.useState("calories");
+    const [selected, setSelected] = React.useState([]);
+    const [isLoading, setIsLoading] = React.useState(true);
+
+    const currentYear = new Date().getFullYear();
+    const currentMonth = new Date().getMonth() + 1;
+
+    const handleChangeDate = (month, year) => {
+        fetchMonth(month, year);
+    };
+
+    const handleRequestSort = (event, property) => {
+        const isAsc = orderBy === property && order === "asc";
+        setOrder(isAsc ? "desc" : "asc");
+        setOrderBy(property);
+    };
+
+    const handleSelectAllClick = event => {
+        if (event.target.checked) {
+            const newSelecteds = rows.map(row => row.id);
+            setSelected(newSelecteds);
+            return;
+        }
+        setSelected([]);
+    };
+
+    const handleClick = (event, id) => {
+        const selectedIndex = selected.indexOf(id);
+        let newSelected = [];
+
+        if (selectedIndex === -1) {
+            newSelected = newSelected.concat(selected, id);
+        } else if (selectedIndex === 0) {
+            newSelected = newSelected.concat(selected.slice(1));
+        } else if (selectedIndex === selected.length - 1) {
+            newSelected = newSelected.concat(selected.slice(0, -1));
+        } else if (selectedIndex > 0) {
+            newSelected = newSelected.concat(
+                selected.slice(0, selectedIndex),
+                selected.slice(selectedIndex + 1),
+            );
+        }
+
+        setSelected(newSelected);
+    };
+
+    const isSelected = id => selected.indexOf(id) !== -1;
+
+    const fetchMonth = async (month, year) => {
+        setIsLoading(true);
+        rows = [];
+
+        const db = firebase.firestore();
+        const response = await db
+            .collection("months")
+            .where("month", "==", month)
+            .where("year", "==", year)
+            .get();
+        rows = response.docs.map(doc => {
+            const row = doc.data();
+            row.id = doc.id;
+            initRow(row);
+            return row;
+        });
+        setIsLoading(false);
+    };
+
+    const initRow = row => {
+        Constants.columns.forEach(cell => {
+            if (cell.numeric) {
+                row[cell.id] = getTotalHoursPerCategory(row.days, cell.id);
+            }
+        });
+    };
+
+    const getTotal = column => {
```

Comment:
Bovenstaande comment geldt ook voor deze functie, die gebruiken we in HoursFooter ook.

Reply:
Done